content/code-security/code-scanning/integrating-with-code-scanning/sarif-support-for-code-scanning.md

```diff
@@ -313,86 +310,6 @@ This SARIF output file has example values to show the minimum required propertie
 }
 ```
 
-### Relative URI Guidance for SARIF Producers
```

Comment:
@akordowski This content was introduced via another recently merged PR - https://github.com/github/docs/pull/35043

Can we ensure this content remains intact? 💛

Reply:
I restored the content and updated the branch.

Reply:
Thank you so much! 💛